### PR TITLE
Using asciibinder for Aerogear.org 

### PR DESCRIPTION
**Pull request by pwright**

### CODE REVIEW

File: aerogear-docs.adoc
Comment (david-martin) [inline aerogear-docs.adoc]: Minor grammar thing here.
'Each release branch be' => 'Each release branch will/should be'

**Comment by david-martin** [comment]
I like this idea.
If the retrieval of docs from x many other repos can be kept automated, that would be great.

Maybe outside the scope of this proposal, but should we be thinking about how search will work on the site?

File: aerogear-docs.adoc
Comment (pwright) [reply aerogear-docs.adoc]: thanks, it was a bit early for 'write like a pirate' day

File: aerogear-docs.adoc
Comment (wtrocki) [inline aerogear-docs.adoc]: `Aerogear.org modular documentation` seems to be better tittle ;)

File: aerogear-docs.adoc
Comment (wtrocki) [inline aerogear-docs.adoc]: @finp I love this proposal! Really good job. 
We added some simple template to follow (mostly headers) that will help with verification and approving this proposal. See: https://github.com/aerogear/proposals/blob/master/template.md

**Comment by wtrocki** [comment]
It think is worth to mention that proposal it's only to cover documentation for services. 
Other website elements are not covered.

**Comment by david-martin** [comment]
@finp We have something tangible now with https://github.com/aerogear/service-docs.
This is a single repo for anything related to the mobile services that end users (Mobile/Mobile Backend developers) would find useful.

I'd imagine there'd be other docs sources for things like UI guides, CLI installation & guides, IDE plugin installation & guides in the future too.

Is this OK with you?
IS there anything needing updating in the proposal based on this approach?

**Comment by pwright** [comment]
@david-martin  I see the content in services-docs as being orphaned for now. We'll need a lot more context and structure before publication, but I don't see anything to block progress. I wonder where docs related to mobile-core will live? another repo? would a mobile-docs repo make sense for all generic material?

**Comment by david-martin** [comment]
> would a mobile-docs repo make sense for all generic material?

Do you mean move existing content from service-docs to a new metrics-docs repo?
I don't mind if that would make things easier for you.

Approving this PR as is though

**Review by david-martin — APPROVED** [review]

**Comment by david-martin** [comment]
Good to merge if we get another +1 or approval

**Review by wtrocki — APPROVED** [review]
Love idea. Demo looks great. 
Proposal is properly drafted and covers main issues.

**Comment by wei-lee** [comment]
+1

**Review by wei-lee — APPROVED** [review]

**Pull request merged**
